portfolio/src/main/webapp/script.js

```diff
@@ -22,6 +21,47 @@ var counter = 1;
 var intervalLength = setInterval(change, 2000);
 document.getElementById('commentBtn').addEventListener('click', getComments);
 
+var script = document.createElement('script');
+script.src = `https://maps.googleapis.com/maps/api/js?key=${MAPKEY}&callback=initMap`;
+script.defer = true;
+script.async = true;
+
+window.initMap = function() {
+    const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422403, lng: -122.088073}, zoom: 5});
+    addLandmark(
```

Comment:
I would put all these datapoints in a list of named variables somehow and call addLandmark in a loop. Right now the data is embedded in the code, and generally you want to separate data and code.